analysis/tests/src/expected/Completion.res.txt

```diff
@@ -1597,14 +1599,8 @@ Resolved opens 3 Completion.res Completion.res pervasives
 ContextPath Value[withCallback](~a)
 ContextPath Value[withCallback]
 Path withCallback
-Found type for function (~b: int) => int
-[{
-    "label": "b",
-    "kind": 4,
-    "tags": [],
-    "detail": "int",
-    "documentation": null
-  }]
+Found type for function int
+[]
```

Comment:
This is right in uncurried mode I believe. The code is:
```rescript
type callback = (~a: int) => int

let withCallback: (~b: int) => callback = (~b) => { (); (~a) => a + b }
```

And the test is:
```rescript
// withCallback(~a)(~
//                                 ^com
```

In curried mode I guess the returned callback is "merged" into the main function definition, so `~a` works even if it's in the _callback_ and not the main function. but in uncurried it's required to explicitly call `withCallback` with the right parameter `~b` before you get the callback back and can invoke that.

Reply:
Indeed the two are not equivalent in uncurried mode.